components/vf-summary/vf-summary--profile.hbs

```diff
@@ -1,5 +1,5 @@
 <article class="vf-summary vf-summary--profile">
-  <img class="vf-summary__image vf-summary__image--avatar" src="{{ path '/assets/vf-summary/assets/vf-summary--profile-img.png' }}" alt="">
+  <img class="vf-summary__image vf-summary__image--avatar" src="{{ path '/assets/vf-summary/assets/vf-icon--avatar.svg' }}" alt="">
```

Comment:
in the hope that we can remove icon fonts (because they're ultimately bad) produced this naming convention.